zcutil/build.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 set -eu -o pipefail
+set +x
```

Comment:
This was originally `set -x`, but given that this script already has `set -x` further down, and can now be called from another script that itself has `set -x`, I think this provides the intended behaviour.